### ISSUE TITLE
sys:1: RuntimeWarning: coroutine 'Launcher.killChrome' was never awaited

### ISSUE THREAD

**Issue by seriaati** [pyppeteer/pyppeteer#400]
the above error message shows when I try to interrupt the script using ctrl+c
I am running a discord bot.

**Comment by redraskal** [comment]
> the above error message shows when I try to interrupt the script using ctrl+c
> 
> I am running a discord bot.

Yeah it looks like the browser is not cleaned up (stays running as a separate process in background) unless you are able to call the close method

**Comment by redraskal** [comment]
Pyppeteer should automatically close the browser on exit but that does not seem to be the case on Windows using CTRL+C
https://github.com/pyppeteer/pyppeteer/blob/3f6ce818f4baa2d02d13c7af15ef717238f336b5/pyppeteer/launcher.py#L151-L165

```
ERROR: This event loop is already running
RuntimeError: Event loop is closed
sys:1: RuntimeWarning: coroutine 'Launcher.killChrome' was never awaited
```

**Comment by seriaati** [comment]
interesting, are there any workarounds?

**Comment by ddurham2** [comment]
Isn't launcher.py just wrong?  it's trying to run another event loop from an already async function, which will make python complain every time.
